compositor_common/src/error.rs

```diff
@@ -44,6 +44,23 @@ impl Display for UnusedNodesError {
     }
 }
 
+#[derive(Debug, thiserror::Error, PartialEq, Eq)]
+pub enum UnsatisfiedConstraintsError {
+    #[error(
+        "Invalid input pads specification for node \"{}\". {} requires {}. {}",
```

Comment:
I think we decided that we don't want to chain error like this, and we are using stacktrace-like approach. See my comment https://github.com/membraneframework/video_compositor/pull/151#discussion_r1324367394